runtime/js-compute-runtime/host_interface/component/fastly_world_adapter.cpp

```diff
@@ -785,6 +785,36 @@ bool fastly_compute_at_edge_dictionary_get(fastly_compute_at_edge_dictionary_han
   return true;
 }
 
+bool fastly_compute_at_edge_config_store_open(fastly_world_string_t *name,
+                                              fastly_compute_at_edge_config_store_handle_t *ret,
+                                              fastly_compute_at_edge_types_error_t *err) {
+  return convert_result(
+      fastly::config_store_open(reinterpret_cast<char *>(name->ptr), name->len, ret), err);
+}
+
+bool fastly_compute_at_edge_config_store_get(fastly_compute_at_edge_config_store_handle_t h,
+                                             fastly_world_string_t *key,
+                                             fastly_world_option_string_t *ret,
+                                             fastly_compute_at_edge_types_error_t *err) {
+  ret->val.ptr = static_cast<uint8_t *>(cabi_malloc(CONFIG_STORE_ENTRY_MAX_LEN, 1));
```

Comment:
I couldn't see offhand directly where `CONFIG_STORE_ENTRY_MAX_LEN` is defined. Is it definitely set correctly?

Reply:
Yep, it's in `fastly.h`:

https://github.com/fastly/js-compute-runtime/blob/1c21c9a7e1ee484c2fb35d614fcda8d373822307/runtime/js-compute-runtime/host_interface/fastly.h#L21

and

https://github.com/fastly/js-compute-runtime/blob/1c21c9a7e1ee484c2fb35d614fcda8d373822307/runtime/fastly/host-api/fastly.h#L21